co-simulation/pom.xml

```diff
@@ -486,6 +486,7 @@
                             <exclude>target/**</exclude>
                             <exclude>**/ClientServerChannelProtos.java</exclude>
                             <exclude>fed/mosaic-carma/**</exclude>
+                            <exclude>fed/mosaic-carla/**</exclude>
```

Comment:
So this line is to exclude carla ambassador from license check. If we do this, why is the missing license file necessary?

Reply:
This was a part of the libpython-clj dependency, after excluding this we still have the licensing issue showing up thats why we added the missing-license file, however this was supposed to be removed since we are not using libpython-clj anymore, sorry for the confusion